src/views/pages/product/single.twig

```diff
@@ -296,6 +296,8 @@
                 {% endif %}
                 {% hook 'product:single.description' %}
 
+                <salla-loyalty-banner data-product-id="{{ product.id }}"></salla-loyalty-banner>
```

Comment:
make sure that there is settings passed to the twilight to avoid running unwanted request

Reply:
@jalmatari Can you please elaborate?

Reply:
as we agreed @maalfaifi